.github/workflows/brim-release.yml

```diff
@@ -2,8 +2,8 @@ name: Brim release
 
 on:
   push:
-    tags:
-      - v*brim*
+    branches:
```

Comment:
Just a reminder to revert this before merging.

Reply:
@henridf. Reverted. Thanks for the reminder.